exercises/practice/resistor-color-trio/src/main/java/ResistorColorTrio.java

```diff
@@ -0,0 +1,12 @@
+class ResistorColorTrio {
+    private enum Color{
+
+    }
+    public static long getValueOfResistor(Color[] colors){
```

Comment:
I'm not sure the class signature makes a lot of sense like this. I'd rather see a single method that takes an array of colors and returns a string with the value and the correct unit:

``` java
new ResistorColorTrio().getValue(new String[] { "orange", "orange", "black" });
// => returns "33 ohms"
```